.travis.yml

```diff
@@ -15,7 +16,8 @@ env:
   - CHEF_VERSION=12.4.1
   - CHEF_VERSION=12.3.0
   - CHEF_VERSION=12.2.1
-  - CHEF_VERSION=12.1.2
+  # - CHEF_VERSION=12.2.0
+  # - CHEF_VERSION=12.1.2    # known broken for now.
```

Comment:
Did the `allow_failures` block not work for these?  I would think we still want it testing and showing up as failures (unless its taking a lot of time or something)